Spanner/src/Operation.php

```diff
@@ -119,13 +119,16 @@ public function deleteMutation($table, KeySet $keySet)
      *     Configuration options.
      *
      *     @type string $transactionId The ID of the transaction.
+     *     @type bool $returnCommitStats If true, return the full response.
+     *           **Defaults to** `false`.
      * }
-     * @return Timestamp The commit Timestamp.
+     * @return Timestamp|array The commit Timestamp or the entire response.
```

Comment:
I understand the motivation here, but we should avoid variable return types if at all possible. Take a look at [this gist](https://gist.github.com/jdpedrie/0d572a00e948e02359619a297089e605) for an idea. This also has the beneficial effect of not forcing us to do the timestamp conversion twice.

Reply:
That is a lot cleaner! Done.